rclrs_examples/CMakeLists.txt

```diff
@@ -56,13 +61,13 @@ ${_native_libraries_dirs}
 file(COPY "${CMAKE_SOURCE_DIR}/Cargo.toml" DESTINATION "${CMAKE_BINARY_DIR}/")
 file(APPEND "${CMAKE_BINARY_DIR}/Cargo.toml" "${_crates_dependencies}")
 
-ExternalProject_Add(
-    rclrs_examples
-    DOWNLOAD_COMMAND ""
-    CONFIGURE_COMMAND ""
-    BUILD_COMMAND cargo build --release --manifest-path "${CMAKE_BINARY_DIR}/Cargo.toml"
-    INSTALL_COMMAND ""
-    LOG_BUILD ON)
+add_custom_target(cargo_build ALL
+    COMMENT "build crate")
+    
+add_custom_command(
```

Comment:
To ensure that this command is only run when the output files are missing or out of date, you can add the following:

```cmake
add_custom_command(
OUTPUT
${CMAKE_BINARY_DIR}/ament_cargo/${PROJECT_NAME}/target/release/rclrs_publisher
${CMAKE_BINARY_DIR}/ament_cargo/${PROJECT_NAME}/target/release/rclrs_subscriber
TARGET cargo_build
...
)
```

Reply:
Apparently OUTPUT and TARGET cannot be mixed in add_custom_command.
> add_custom_command Wrong syntax.  A TARGET and OUTPUT can not both be specified.

Without TARGET I can't manage to make CMake trigger rebuild on source change :disappointed: